app/templates/components/infinity-loader.hbs

```diff
@@ -1,7 +1,7 @@
 {{#if hasBlock}}
-  {{yield}}
+  {{yield infinityModelContent}}
```

Comment:
I'm sure this is already on your radar, but some schweet documentation improvements would be awesome!

Reply:
100% - wanted to get your thoughts first!  Next I'll hit tests, then docs